storage/storage.go

```diff
@@ -348,3 +395,205 @@ func EncodeName(name string) string {
 	}
 	return result
 }
+
+func SeparateMod(ctx context.Context, body []byte, modID, name string, versionID string, modVersion string) (bool, string) {
+	//read combined file
+	zipReader, err := zip.NewReader(bytes.NewReader(body), int64(len(body)))
+
+	if err != nil {
+		return false, ""
+	}
+
+	// Create a buffer to write our archive to.
+	log.Ctx(ctx).Info().Msg("Beginning Splitting Operations")
```

Comment:
Please add more context to the log e.g. the mod id and version

Reply:
Removed as it was used for when I was first splitting so I had a log of where I was...
https://github.com/satisfactorymodding/smr-api/pull/12/commits/0938ddafb99e9ba28c8d21ee0c6a45f34d7be867